### DISCUSSION TITLE
Welcome to Briefcase!

### DISCUSSION THREAD

**Discussion by freakboy3742** [beeware/briefcase#585] [General]
## 👋 Welcome!

Briefcase is part of the [BeeWare Project](https://beeware.org).

This discussion forum is a place where you can:
* Ask questions about how to use Briefcase to package the app of your dreams
* Discuss suggestions for new features, or changes to existing features
* Share details about projects that you've packaged with Briefcase
* ...and anything else related to Briefcase that needs to be discussed!

We want this to be a welcoming space for all, so this server (along with all BeeWare community spaces) is subject to the [BeeWare Community Code of Conduct](https://beeware.org/community/behavior/).

If you want to say "Hi!", comment below with an introduction of yourself and tell us about what you do with Briefcase.

**Comment by RasulAramian** [comment]
Hi, i have a problem with this command "briefcase create android"; 
![Screenshot from 2023-07-30 12-16-12](https://github.com/beeware/briefcase/assets/44385827/d43c7a30-b9b7-42d2-b529-146be3707bd8)

**Reply by mhsmith** [reply]
Your user page says you're in Iran. Unfortunately [many services block Iranian IP addresses because of US sanctions](https://medium.com/@AliBehzadian/pain-of-being-android-developer-in-iran-390fb3190cb3). Try using a VPN.

**Reply by RasulAramian** [reply]
Thank you for your reply.
